wpeverest/user-registration
# Enhance/search fields

### All Submissions:






### Changes proposed in this Pull Request:
This PR adds the "Search Fields" feature.

### How to test the changes in this Pull Request:

1. Create a new form
2. Go to fields tab and type some field name in the search box
3. Only the matching fields should appear

### Types of changes:




### Other information:




### Changelog entry

> Enhance - Add "Search Fields" feature.
